parentchain/src/tests.rs

```diff
@@ -0,0 +1,56 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+use crate::mock::*;
+use frame_support::{assert_err, assert_ok, sp_runtime::traits::Header as HeaderT};
+use sp_core::H256;
+use sp_keyring::AccountKeyring;
+use sp_runtime::{generic, traits::BlakeTwo256, DispatchError::BadOrigin};
+
+pub type Header = generic::Header<BlockNumber, BlakeTwo256>;
+
+#[test]
+fn verify_storage_works() {
+	let block_number = 3;
+	let parent_hash = H256::from_low_u64_be(420);
+
+	let header: Header = HeaderT::new(
```

Comment:
Did you notice that here you're using the Header Trait (`HeaderT::new()`) from `frame_support::sp_runtime`, whereas in the `non_root_account_errs` fn you're using `Header::new()` (i.e. the Header imported from `sp_runtime::generic`). Not that I mind, as it's for testing only 😋